### PR TITLE
Add IOPs to volume

### PR DESCRIPTION
**Pull request by michaelkad**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
=== RUN   TestAccIBMPIVolumeDataSource_basic
--- PASS: TestAccIBMPIVolumeDataSource_basic (16.46s)
PASS

...
```
https://jsw.ibm.com/browse/POWERIAAS-9894

### CODE REVIEW

**Review by ismirlia — DISMISSED** [review]

File: ibm/service/power/data_source_ibm_pi_volume.go [hunk 1]
```diff
@@ -145,6 +150,7 @@ func dataSourceIBMPIVolumeRead(ctx context.Context, d *schema.ResourceData, meta
 	d.Set("primary_role", volumedata.PrimaryRole)
 	d.Set("auxiliary_volume_name", volumedata.AuxVolumeName)
 	d.Set("master_volume_name", volumedata.MasterVolumeName)
+	d.Set("io_throttle_rate", volumedata.IoThrottleRate)
```
Comment (yussufsh) [inline ibm/service/power/data_source_ibm_pi_volume.go:153]: Is it set for all the volumes by default? If not then this test fail for some volumes.

Comment (michaelkad) [reply ibm/service/power/data_source_ibm_pi_volume.go:153]: You’re correct. I update the test case

File: website/docs/d/pi_volume.html.markdown [hunk 2]
```diff
@@ -60,3 +60,4 @@ In addition to all argument reference list, you can access the following attribu
 - `state` - (String) The state of the volume.
 - `volume_pool` - (String) Volume pool, name of storage pool where the volume is located.
 - `wwn` - (String) The world wide name of the volume.
+- `io_throttle_rate` -(string) Amount of iops assigned to the volume.
```
Comment (yussufsh) [inline website/docs/d/pi_volume.html.markdown:63]:
```suggestion
- `io_throttle_rate` - (String) Amount of iops assigned to the volume.
```

Comment (michaelkad) [reply website/docs/d/pi_volume.html.markdown]: Done.

**Pull request closed without merging**